torrent_accessors.go

```diff
@@ -148,6 +148,7 @@ type Torrent struct {
 	IsFinished              *bool              `json:"isFinished"`
 	IsPrivate               *bool              `json:"isPrivate"`
 	IsStalled               *bool              `json:"isStalled"`
+	Labels                  []string           `json:"labels"`
```

Comment:
Could you add add a comment specifying this is a RPC v16 field ?

Reply:
Done